underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -452,7 +453,7 @@ private ObjectListing getObjectListingChunkV1(ListObjectsRequest request) throws
     @Override
     public String[] getCommonPrefixes() {
       List<String> res = mResult.getCommonPrefixes();
-      return res.toArray(new String[res.size()]);
+      return res.toArray(new String[0]);
```

Comment:
Based on https://stackoverflow.com/questions/18136437/whats-the-use-of-new-string0-in-toarraynew-string0
looks like it will be better to create the array with new String[size] than new String[0]

Reply:
Actually in the comments of this answer, there's a link https://stackoverflow.com/questions/174093/toarraynew-myclass0-or-toarraynew-myclassmylist-size/29444594#29444594 that says giving `new String[0]` is consistently faster. Plus Intellij throws a warning if you give it a pre-sized array.